### PR TITLE
i did a lot of things

### PR DESCRIPTION
**Pull request by notmarkmiranda**
- js true, selenium testing for new projects form.
- changed votes.count to take into account for 0 votes (divide by zero error)
- database cleaner, test was failing because of finding a user that was an admin for non-admin test

### CODE REVIEW

**Comment by notmarkmiranda** [comment]
@s-espinosa hi

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
   def new
-    @project = Project.new
-    @modules = ["BE Mod 2", "BE Mod 3", "BE Mod 4", "FE Mod 2", "FE Mod 3", "FE Mod 4", "Posse"]
+    if DemoNight.current
+      @project = Project.new
+      @current = DemoNight.current.id
+      @modules = ["BE Mod 2", "BE Mod 3", "BE Mod 4", "FE Mod 2", "FE Mod 3", "FE Mod 4", "Posse"]
+    else
+      flash[:error] = "There is no active demo night, please ask Sal to make one"
```
Comment (s-espinosa) [inline app/controllers/projects_controller.rb]: This is my favorite flash message ever.

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+      redirect_to root_path
+    end
   end
 
   def create
-    @project = DemoNight.current.projects.new(project_params)
+    @project = Project.new(project_params)
```
Comment (s-espinosa) [inline app/controllers/projects_controller.rb]: Might change this back, but works for now. Any reason we can't leave as `DemoNight.current.projects.new(project_params)`?

File: spec/features/user_can_create_a_project_spec.rb [hunk 3]
```diff
@@ -9,10 +9,11 @@
     visit new_project_path
     fill_in "project[group_members]", with: "Sharon Jones"
     fill_in "project[name]", with: "Witty Name"
-    select "BE Mod 3", from: "project[project_type]"
-    check "project[final_confirmation]"
+    # select "BE Mod 3", from: "project_project_type"
```
Comment (s-espinosa) [inline spec/features/user_can_create_a_project_spec.rb]: Comments! In your code! Good day, sir!

**Pull request merged**